zproject/prod_settings_template.py

```diff
@@ -717,6 +717,7 @@
 ## path.  For that reason, we recommend replacing /home/zulip/uploads
 ## with a symlink instead of changing LOCAL_UPLOADS_DIR.
 LOCAL_UPLOADS_DIR = "/home/zulip/uploads"
+# S3_SKIP_PROXY = False
```

Comment:
Why this position in the sort order?  I'd prefer to put it after `S3_ENDPOINT_URL` from a "what everyone needs" standpoint.

Reply:
Sure.